app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Mappings.scala

```diff
@@ -25,38 +25,44 @@ import java.time.LocalDate
 
 trait Mappings extends Formatters with Constraints {
 
-  protected def text(errorKey: String = "error.required", args: Seq[String] = Seq.empty): FieldMapping[String] =
-    of(stringFormatter(errorKey, args))
+  protected def sanitise(value: String) =
+    value.replaceAll("\\s", "")
```

Comment:
`str.filterNot(_.isWhitespace)`

this is a functional way to do it

Reply:
Changed the code.